modules/http-load-balancer/variables.tf

```
@@ -34,6 +34,12 @@ variable "ssl_certificates" {
   default     = []
 }
 
+variable "ssl_policy" {
+  description = "Link to ssl policy. Omitted if not provided"
```

```suggestion
  description = "A reference to the SslPolicy resource that will be associated with the TargetHttpsProxy resource. If not set, the TargetHttpsProxy resource will not have any SSL policy configured."
```
We'll adopt the TF provider docs as its clearer to the users.